pkg/csi/cinder/openstack/openstack.go

```diff
@@ -125,7 +125,8 @@ func CreateOpenStackProvider() (IOpenStack, error) {
 	}
 
 	epOpts := gophercloud.EndpointOpts{
-		Region: cfg.Global.Region,
+		Region:       cfg.Global.Region,
+		Availability: cfg.Global.EndpointType,
```

Comment:
Type Conversion missing here?

Reply:
no, it is not, since both `Availability` and `cfg.Global.EndpointType` are `gophercloud.Availability` https://github.com/gophercloud/gophercloud/blob/dee9bb4c60eccb45e80e3d2d0f4ced546f421ad6/endpoint_search.go